src/components/SendComponentWithData.js

```diff
@@ -80,7 +80,7 @@ export const withSendComponentWithData = (SendComponent, sendTransactionCb) => {
     };
 
     state = {
-      isValidAddress: true,
+      isValidAddress: false,
```

Comment:
@jinchung - I just wanted a sanity check about the `isValidAddress`.

The value no longer defaults to true, and as the value of `recipient` is updated, `isValidAddress` is updated.

Reply:
good catch - makes sense, thanks!